projects/admin/src/app/record/detail-view/organisation-detail-view/organisation-detail-view.component.html

```diff
@@ -41,7 +41,10 @@ <h1>{{ record.metadata.name }}</h1>
     <!-- ACTIVE BUDGET -->
     <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title">Active budget</dt>
     <dd class="col-sm-7 col-md-8 mb-0">
-      <admin-budget-select [organisation]="record"></admin-budget-select>
+      <admin-budget-select *ngIf="isSystemLibrarian" [organisation]="record"></admin-budget-select>
+      <ng-container *ngIf="!isSystemLibrarian">
```

Comment:
Wouldn't this be better with ng-template and elseBlock?

Reply:
done